packages/corelib/src/dataModel/RundownPlaylist.ts

```diff
@@ -69,20 +69,18 @@ export interface DBRundownPlaylist {
 	metaData?: unknown
 
 	/** the id of the Live Part - if empty, no part in this rundown is live */
-	// currentPartInstanceId: PartInstanceId | null
 	currentPartInfo: SelectedPartInstance | null
 	/** the id of the Next Part - if empty, no segment will follow Live Part */
-	// nextPartInstanceId: PartInstanceId | null
 	nextPartInfo: SelectedPartInstance | null
 	/** The time offset of the next line */
 	nextTimeOffset?: number | null
-	/** if nextPartId was set manually (ie from a user action) */
-	nextPartManual?: boolean
```

Comment:
Personally, I think this should be mentioned in the migrations, even its a "temporary" datapoint (for consistency).